docker/README.rst

```diff
@@ -89,15 +93,51 @@ Next, install ``curl`` by opening an Ubuntu terminal then typing:
 
     sudo apt-get install -y curl
 
-2. Run the bootstrap script
-***************************
+2. Download the bootstrap script
+********************************
+To keep your filesystem tidy, create a directory before running the following commands, since these commands download several scripts and subdirectories. To do this:
+
+    .. code-block:: bash
+
+        mkdir runestone
+        cd runestone
+
 Next, download the bootstrap script. To do this, open a terminal in Ubuntu or OS X then type:
 
 .. code-block:: bash
 
     curl -fLO https://raw.githubusercontent.com/RunestoneInteractive/RunestoneServer/master/docker/docker_tools.py
 
-This download the bootstrap script. The next step, which installs required dependencies for the remainder of the process, depends on the two mutually exclusive use cases below. **Remember which use case you select**; many of the following steps vary based on your use case.
+This download the bootstrap script.
+
+
+3. Create then activate a Python virtual environment
+****************************************************
+A Python virtual environment ensures that all the Python dependencies installed by this process don't interfere with your global / system Python installation.
+
+#.  `Create a Python virtual environment <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment>`_ named ``runestone``:
+
+    .. code-block:: bash
+
+        python3 -m venv runestone
+
+#.  `Activate the virtual environment <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#activating-a-virtual-environment>`_ you just created.
+
+    On Windows:
+
+        .. code-block:: bash
+
+            runestone\Scripts\activate
```

Comment:
maybe rsve is a more descriptive, less confusing name than putting the runestone virtual environment in runestone especially when there are so many other folder names that are either runestone or start with runestone?

Reply:
Good point -- let me think of a more descriptive name...

Reply:
I'm using `rsvenv` instead.